### PR TITLE
docs: Add instructions to migrate to Mbed CLI 2

### PR DESCRIPTION
**Pull request by urutva**
Starting with version 6.5, Mbed OS  uses Mbed CLI 2. It uses Ninja as a build system, and CMake to generate the build environment and manage the build process in a compiler-independent manner.

### CODE REVIEW

**Review by 0xc0170 — APPROVED** [review]
This would be good to propagate to all examples, shall I assist to coordinate?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
 ![](./resources/official_armmbed_example_badge.png)
 # Blinky Mbed OS example
 
-The example project is part of the [Arm Mbed OS Official Examples](https://os.mbed.com/code/) and is the [getting started example for Mbed OS](https://os.mbed.com/docs/mbed-os/v5.14/quick-start/index.html). It contains an application that repeatedly blinks an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
+The example project is part of the [Arm Mbed OS Official Examples](https://os.mbed.com/code/) and is the [getting started example for Mbed OS](https://os.mbed.com/docs/mbed-os/latest/quick-start/index.html). It contains an application that repeatedly blinks an LED on supported [Mbed boards](https://os.mbed.com/platforms/).
```
Comment (Patater) [inline README.md:4]: This quick start guide doesn't cover Mbed CLI 2 yet.

File: README.md [hunk 2]
```diff
@@ -21,22 +30,26 @@ The `main()` function is the single thread in the application. It toggles the st
...
+
     ```bash
     $ mbed compile -m <TARGET> -t <TOOLCHAIN> --flash
     ```
 
 Your PC may take a few minutes to compile your code.
 
-The binary is located at `./BUILD/<TARGET>/<TOOLCHAIN>/mbed-os-example-blinky.bin`.
+The binary is located at: </br>
+**Mbed CLI 2** - `./cmake_build/mbed-os-example-blinky.bin`</br>
```
Comment (Patater) [inline README.md:48]: I'd recommend using an unordered list, instead of manual line breaks.

**Review by Patater — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (urutva) [reply README.md:4]: This will be fixed in a new PR.

File: README.md (hunk 2 above)
Comment (urutva) [reply README.md]: Fixed.

**Review by Patater — APPROVED** [review]

**Comment by 0xc0170** [comment]
I'll merge this as it contains docs update to support new Mbed CLI 2.

**Comment by urutva** [comment]
@0xc0170 I'll close this PR and open another one for `development`.

**Pull request closed without merging**